src/z3c/authviewlet/auth.py

```diff
@@ -93,9 +90,9 @@ def available(self):
         return not authenticated(self.request.principal)
 
     def render(self):
-        return u'<a href="%s?nextURL=%s">%s</a>' % (
+        return '<a href="{}?nextURL={}">{}</a>'.format(
             get_view_url(self. context, self.request, self.viewName),
```

Comment:
I wonder how this ever worked with the blank space there 

```suggestion
            get_view_url(self.context, self.request, self.viewName),
```